R/ivreg_marg.R

```diff
@@ -0,0 +1,124 @@
+#' Predicted Margins for `ivreg` objects from the \code{AER} package
+#'
+#' Obtains predicted margins and standard errors
+#' of those predictions from a fitted \code{ivreg} model object.
+#'
+#' @inheritParams marg
+#' @param ... additional parameters passed to \code{?marg}.
+#'
+#' @export
+#' @examples
+#' data(margex)
```

Comment:
Why doesn't this example just load in some data from the `AER` package instead of going to such lengths to build our own?